package.json

```diff
@@ -108,8 +108,8 @@
     "jest-react-native": "^18.0.0",
     "prettier": "^1.11.1",
     "react-native-cli": "^2.0.1",
-    "reactotron-react-native": "^2.0.0-beta.1",
-    "reactotron-redux": "^2.0.0-beta.1",
+    "reactotron-react-native": "^1.14.0",
+    "reactotron-redux": "^1.13.0",
```

Comment:
Reverting to previous more stable version of reactotron.

Reply:
What issues did you experience?

Reply:
Some of my api request's body were not getting logged. Also the order of actions being fired was changed from the original order. Those little pesky quirks are still there So i think we should wait for v2 to release in stable version and then migrate.